tests/snapshots/tests.contrib.grpc.test_grpc.test_method_service.json

```diff
@@ -19,6 +19,7 @@
       "grpc.status.code": "StatusCode.OK",
       "language": "python",
       "network.destination.port": "58143",
+      "rpc.service": "Servicer",
```

Comment:
could you explain Servicer vs pkg.Servicer and how we gain from these distinct values?

Reply:
This test looks at the trace produced by 2 rpc methods calls: `/Servicer/Handler/` and `/pkg.Servicer/Handler/`.

The span modified here relates to the first call, so we don't have package information, which is expected - there's no duplication of purpose here.

Re: `grpc.method.service` versus `rpc.service`, the purpose of `rpc.service` is to contain package (if available) _and_ service, such that the tag provides a relevant unit of service that can be used by the service map. The end goal is for the new tag to be a precursor of the upcoming `peer.service` tag, which will become the new source of truth for the service map.

Reply:
Ah got it, took a slight mental shift. Thanks for the explanation!